@here/harp-map-theme/resources/berlin_tilezen_base.json

```diff
@@ -1100,7 +1085,8 @@
             ["ref", "waterPolygons"],
             {
                 "description": "water",
-                "when": ["==", ["get", "$layer"], "water"],
+                "layer": "water",
+                "when": ["has", "name"],
```

Comment:
our `when` conditions are mandatory so I need to put something here. Is the above condition correct, or should I use something else?

Reply:
If we have `layer` then `when` should be conditional, why not making it so ?